### PR TITLE
Fix the placement of licenses

### PR DESCRIPTION
**Pull request by Serial-ATA**
Signed-off-by: Serial <69764315+Serial-ATA@users.noreply.github.com>

You had some licenses between imports, or missed some newlines :)

### CODE REVIEW

File: src/ui/components/key_editor/ke_input.rs [hunk 1]
```diff
@@ -1,9 +1,7 @@
 //! # Popups
 //!
-//!
 //! Popups components
```
Comment (Serial-ATA) [inline src/ui/components/key_editor/ke_input.rs:3]: Can the doc comments like these be removed? I don't think they add much value, and they prevent the licenses from being at the top of the file.

**Comment by Serial-ATA** [comment]
Also, unrelated to these changes, there were a lot of commented out imports. Should they be removed? I noticed a few were noted to be for debugging, but most can probably be taken out.

File: src/ui/components/key_editor/ke_input.rs (hunk 1 above)
Comment (tramhao) [reply src/ui/components/key_editor/ke_input.rs:3]: Yes, sure.

**Comment by tramhao** [comment]
Yes, thanks for your help:)

**Pull request merged**